ui/src/components/LanguageDropdown/index.js

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
-import { Menu, Icon, Dropdown } from 'antd';
+import { Menu, Dropdown } from 'antd';
 import _ from 'lodash';
 import { withTranslation } from 'react-i18next';
-import NavAction from './NavAction';
 
-class LangDropdown extends React.PureComponent {
+@withTranslation()
```

Comment:
We can try the function component so we can use `useTranslation()` react hook instead of HOC, I used in https://github.com/pingcap-incubator/tidb-dashboard/commit/97c96ad10cecf342b33dc4d70a2f6a918e0a8ca5